backend/migrations/2023-04-20-151000_base_layer/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE base_layers (
+    id SERIAL PRIMARY KEY,
+    base_image_url TEXT NOT NULL,             -- nextcloud URL providing the base image
```

Comment:
URL or Photoname? So should it be /Photos/Vineyard.jpg or https://cloud.permaplant.net/apps/files/?dir=/Photos&fileid=515787? It didn't work for me in either case. @absurd-turtle expected is that people can select a file from hard disc (to be uploaded to nextcloud) or directly from nextcloud.

Furthermore: when typing in "Background Image File Path" every letter produced an undo step. Expected behavior: the whole change of the image should be undoable, not the exact text I typed.

Reply:
I forgot to delete these scripts, thanks for reminding me.

Reply:
> URL or Photoname?

The first option.

>Furthermore: when typing in "Background Image File Path" every letter produced an undo step. 
> Expected behavior: the whole change of the image should be undoable, not the exact text I typed.

Would it be ok for you if I just add a submitt button for now?

> It didn't work for me in either case.

I just checked and there seems to be a bug in the nextcloud client.
For example, using the default Image Photos/Frog.jpg works:
![image](https://github.com/ElektraInitiative/PermaplanT/assets/27735433/fca7926b-0cb4-488e-953a-7f5f945ab38a)

An image that was uploaded later like this, produces a 404 error from Nextcloud:
![image](https://github.com/ElektraInitiative/PermaplanT/assets/27735433/30822343-ccb6-413d-9ffa-beefb4ab5aeb)

Interestingly, default images are still displayed after I deleted them.
It seems like Nextcloud does not access the correct files for some reason...

Reply:
Ahh, I just see that you didn't enter a `/` in the beginning of the file path. Is this maybe the problem why it didn't work for me? Can you please allow such a slash?

Reply:
Sure, shouldn't be a problem

Reply:
> I just checked and there seems to be a bug in the nextcloud client.

I think I figured out what causes the strange Nextcloud related behavior:

I was uploading/deleteing images on the account that was migrated from markus-raab.org instead of the account associated with my keycloak credentials.
The Nextcloud-Client should to be fine.